config/postBuild.js

```diff
@@ -0,0 +1,4 @@
+const fs = require('fs');
+
+fs.copyFileSync('README.md', 'dist/README.md');

```

Comment:
let's use the modern one here https://nodejs.org/dist/latest-v12.x/docs/api/fs.html#fs_fspromises_copyfile_src_dest_flags.